### DISCUSSION TITLE
SEB Configuration by command line

### DISCUSSION THREAD

**Discussion by hinra** [SafeExamBrowser/seb-win-refactoring#1020] [Q&A]
I would like to know if it is possible to configure a .seb-file by command line. I would like to set up a webpage where non-technical collegues can configure a file with few options. The config-tool has to many option and unknown terms for most of my collegues. The web page will then host links for the students to start SEB by one click. (like seb://myserver.com/test.seb) 

Or if anyone has an idea how achive this by others means. 

Regards
Ralf

**Comment by dbuechel** [comment]
Detailed instructions on how to integrate SEB into a web application can be found on our website: https://safeexambrowser.org/developer/seb-integration.html.

For your particular use case, you could generate a data URL on server side which directly contains the configuration as chosen by your users in the web application.
